le_pr_delivery/delivery/urls.py

```diff
@@ -1,11 +1,14 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
 from . import views
+from delivery.views import *
```

Comment:
aren't the two imports 
 from . import views
from delivery.views import *

basically the same? if it is we should delete one.